src/main.rs

```diff
@@ -177,7 +184,7 @@ fn analyze_and_save_results(
 
     let mut config: Config<DefaultBackend> = Config::default();
     config.null_pointer_checking = config::NullPointerChecking::None; // In the Tock kernel, we trust that Rust safety mechanisms prevent null pointer dereferences.
-    config.loop_bound = 50; // default is 10, raise if larger loops exist
+    config.loop_bound = 1000; // default is 10, raise if larger loops exist
```

Comment:
I think we want to keep this lower so as to more quickly find places in the code where actual loops are causing failures, rather than complexity explosion. If you find places in the code where Haybale gives up because the loop bound is hit, we should modify the Tock code rather than raising this bound.